### PR TITLE
Formated layout for PostView

### PR DESCRIPTION
**Pull request by Emilio22**

### CODE REVIEW

**Comment by Emilio22** [comment]
formatted layout for PostView and created an extension to MediaPost to format the date into a more condensed readable string

File: Birdui/Birdui/PostView.swift [hunk 1]
```diff
@@ -9,20 +9,44 @@
...
+        // The image, if any, is horizontally centered in the view.
+        
+        VStack{
+            HStack {
+                Image("mascot_swift-badge")
+                    .resizable()
+                    .frame(width: 50, height: 50)
+                VStack(alignment: .leading) {
+                    Text("\(post.userName)")
+                    //temp untill I figure out how to format it properly
```
Comment (Zoha131) [inline Birdui/Birdui/PostView.swift]: I think you already figured out the formatting issue. Now you can remove this comment. 🥳

**Comment by Emilio22** [comment]
this should make a pull request without the comment

**Comment by Emilio22** [comment]
PostListView lists all the posts from the ViewModel.  When a new post is added to the model, the view reloads.  Add button will add a media post that I initialized within the action, NewPostView will supply the media post in the future.  The header still needs some tinkering to match the layout specified in the homework pdf.

**Pull request merged**